### PR TITLE
Fix encoding for unit tests and display info

### PR DESCRIPTION
**Pull request by s-hertel**
##### SUMMARY
Fix encoding for unit tests and display info

##### ISSUE TYPE

- Bugfix Pull Request
- Tests Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: lib/ansible/galaxy/collection/__init__.py [hunk 1]
```diff
@@ -997,7 +997,7 @@ def find_existing_collections(path, artifacts_manager):
 
             display.vvv(
                 "Found installed collection {coll!s} at '{path!s}'".
-                format(coll=to_text(req), path=to_text(req.src))
+                format(coll=to_native(req), path=to_native(req.src))
```
Comment (jborean93) [inline lib/ansible/galaxy/collection/__init__.py:1000]: Display should take in text strings not not native strings. We usually leave native for exceptions though.

**Review by jborean93 — COMMENTED** [review]
Having `collection.install` take in a native string for the output path sounds wrong. Usually we try and make sure we are dealing with text strings in all our code and only convert to bytes/native if the function we are then calling at the boundary requires that.

File: lib/ansible/galaxy/collection/__init__.py (hunk 1 above)
Comment (webknjaz) [reply lib/ansible/galaxy/collection/__init__.py:1000]: Right, so maybe having a u-string should be fine.

Comment (jborean93) [reply lib/ansible/galaxy/collection/__init__.py:1000]: Yep, they were incorporated into https://github.com/webknjaz/ansible/pull/357.

**Pull request closed without merging**